### ISSUE TITLE
Works well at first but then stopped working on Redmi Note 8 Pro

### ISSUE THREAD

**Issue by lorograchi** [KieronQuinn/TapTap#167]
It works very well when I installed the app. However, after some time it doesn't work. Tried to change various settings in the app as well as in general like accessibility and battery but of no use. Please fix this. Thank you and I appreciate your work.

**Comment by KieronQuinn** [comment]
#48
